### PR TITLE
Fix 340

### PR DESCRIPTION
**Pull request by markshenouda**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

## Closing issues

closes #ISSUE_NUMBER_GOES_HERE

...

## Description

...

## Changes

- High level
- changes that
- you made
- ...

### CODE REVIEW

File: src/components/DocsNavigation/DocsNavigationMobile/DocsNavigationMobile.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ function DocsNavigationDesktop({ pathname }: { pathname: string; children?: Reac
       >
         <div className={styles.DocsPickerContainer}>
           <DocsPickerMobile path={pathname} />
+          {children}
```
Comment (Zelig880) [inline src/components/DocsNavigation/DocsNavigationMobile/DocsNavigationMobile.tsx:17]: I cannot seem to find any usage of this in the PR. Will look into the code now, but if you know share the reason for this addition

File: src/components/CCIP/Chain/Chain.astro [hunk 2]
```diff
@@ -110,13 +109,13 @@ const searchLanes = getSearchLanes({ environment })
...
     grid-template-columns: 1fr;
-    gap: var(--space-8x);
+    gap: var(--space-2x);
   }
 
   .tokens__grid {
     display: grid;
-    grid-template-columns: 1fr 1fr;
-    gap: var(--space-8x);
+    grid-template-columns: 32% 32% 32%;
```
Comment (Zelig880) [inline src/components/CCIP/Chain/Chain.astro:117]: why use the percentage instead than the FR?

File: src/components/CCIP/Tables/ChainTable.tsx [hunk 3]
```diff
@@ -54,80 +54,82 @@ function ChainTable({ lanes, explorerUrl, sourceNetwork, environment }: TablePro
...
-                    className="ccip-table__network-name"
-                    role="button"
-                    onClick={() => {
-                      const laneData = getLane({
-                        sourceChain: sourceNetwork.key as SupportedChain,
-                        destinationChain: network.key as SupportedChain,
-                        environment,
-                        version: Version.V1_2_0,
-                      })
+      <div className="ccip-table__wrapper">
```
Comment (Zelig880) [inline src/components/CCIP/Tables/ChainTable.tsx:57]: Wrong margin should be 24
image

File: src/components/CCIP/Cards/NetworkCard.css [hunk 4]
```diff
@@ -2,7 +2,7 @@
   display: flex;
   padding: var(--space-6x);
   gap: var(--space-3x);
-
+  width: 100%;
```
Comment (Zelig880) [inline src/components/CCIP/Cards/NetworkCard.css:5]: The main search is broken, Nothing happens when searching.

**Review by Zelig880 — CHANGES_REQUESTED** [review]

File: src/components/CCIP/ChainHero/ChainHero.css [hunk 5]
```diff
@@ -36,7 +36,7 @@
 }
 
 .ccip-chain-hero__content {
-  --gutter: var(--space-10x);
```
Comment (Zelig880) [inline src/components/CCIP/ChainHero/ChainHero.css:39]: Spacing in this file are wrong (not particularly in this line of code).

It should be 24 px between the headings and 8 between label and text:
image

**Review by Zelig880 — CHANGES_REQUESTED** [review]

File: src/components/DocsNavigation/DocsNavigationMobile/DocsNavigationMobile.tsx (hunk 1 above)
Comment (markshenouda) [reply src/components/DocsNavigation/DocsNavigationMobile/DocsNavigationMobile.tsx:17]: To make the filter tabs available on Mobile navbar, previously they were on only Desktop
Screenshot 2024-10-23 at 4 14 02 PM

File: src/components/CCIP/Chain/Chain.astro (hunk 2 above)
Comment (markshenouda) [reply src/components/CCIP/Chain/Chain.astro:124]: Well the grid doesn't respect the page width I tried to force it but it didn't work

Here is the current result
Screenshot 2024-10-23 at 4 23 35 PM

Here is the result with `1fr 1fr 1fr`
Screenshot 2024-10-23 at 4 24 43 PM


So the 32% is the perfect percentage to make the grid looks good in almost every mobile screen size, we are still using 1fr for the desktop

https://github.com/user-attachments/assets/2afab58a-b7de-4e85-859a-bd97df220af3

File: src/components/CCIP/ChainHero/ChainHero.css [hunk 6]
```diff
@@ -54,19 +54,20 @@
 
 .ccip-chain-hero__top {
   display: flex;
+  flex-direction: column-reverse;
```
Comment (Zelig880) [inline src/components/CCIP/ChainHero/ChainHero.css:57]: Not sure if it is this page, but side padding still too small here:
image

File: src/components/CCIP/Chain/Chain.astro (hunk 2 above)
Comment (Zelig880) [reply src/components/CCIP/Chain/Chain.astro:124]: I do not have your results. For me using `fr` works fine:
image

Comment (markshenouda) [reply src/components/CCIP/Chain/Chain.astro]: Oh I tried it as well and it didn't make the same issue, probably it was a problem with the layout and I fixed it while modifying it

Comment (Zelig880) [reply src/components/CCIP/Chain/Chain.astro]: I found out what this is. It is a bug with "display: contents" that is applied when using Astro islands. I reverted it back to 32%

**Review by Zelig880 — APPROVED** [review]

**Pull request merged**